.drone.star

```diff
@@ -214,7 +214,6 @@ def checkForRecentBuilds(ctx):
     return pipelines
 
 def stopRecentBuilds(ctx):
-    repo_slug = ctx.build.source_repo if ctx.build.source_repo else ctx.repo.slug
 
```

Comment:
https://drone.owncloud.com/owncloud/ocis/6272/45/3
```
 def stopRecentBuilds(ctx):
-
     return [{
         "name": "stop-recent-builds",
         "image": "drone/cli:alpine",
```

It wants the empty line deleted.

Reply:
my bad :-( is resolved now